docs/source/using/troubleshooting.adoc

```diff
@@ -194,3 +194,29 @@ flag in the `minishift start` command. Alternatively, the problem might be
 an outdated version of VirtualBox.
 
 To avoid this issue, it is recommended to use VirtualBox 5.1.12 or later.
+
+[[insufficient-privileges]]
+== Hyper-V: Administrator privileges insufficient to run Hyper-V
+
```

Comment:
Awesome. I am fine with how we explain to add oneself to the Hyper-V admin group. It is really that the reasoning why this is required is missing. Effectively this is similar to "sudo" on Linux.